mcc/odin/odin/snapshot_list.go

```diff
@@ -1,12 +1,37 @@
 package odin
 
 import (
+	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/aws/aws-sdk-go/service/rds"
 	"github.com/aws/aws-sdk-go/service/rds/rdsiface"
 )
 
+const (
+	// RFC8601 is the date/time format used by AWS.
+	RFC8601 = "2006-01-02T15:04:05-07:00"
```

Comment:
I know, it's very weird, but I still prefer that than the %d-%m-%YT%H:%M:%S%Z thing... I always need to keep the table of verbs open aside.